statistics/handle/dump_test.go

```diff
@@ -89,49 +138,51 @@ func (s *testStatsSuite) TestDumpGlobalStats(c *C) {
 	tk.MustExec("analyze table t")
 
 	// global-stats is not existed
-	stats := s.getStatsJSON(c, "test", "t")
-	c.Assert(stats.Partitions["p0"], NotNil)
-	c.Assert(stats.Partitions["p1"], NotNil)
-	c.Assert(stats.Partitions["global"], IsNil)
+	stats := getStatsJSON(t, dom, "test", "t")
+	require.NotNil(t, stats.Partitions["p0"])
+	require.NotNil(t, stats.Partitions["p1"])
+	require.Nil(t, stats.Partitions["global"])
 
 	// global-stats is existed
 	tk.MustExec("set @@tidb_partition_prune_mode = 'dynamic'")
 	tk.MustExec("analyze table t")
-	stats = s.getStatsJSON(c, "test", "t")
-	c.Assert(stats.Partitions["p0"], NotNil)
-	c.Assert(stats.Partitions["p1"], NotNil)
-	c.Assert(stats.Partitions["global"], NotNil)
+	stats = getStatsJSON(t, dom, "test", "t")
+	require.NotNil(t, stats.Partitions["p0"])
+	require.NotNil(t, stats.Partitions["p1"])
+	require.NotNil(t, stats.Partitions["global"])
 }
 
-func (s *testStatsSuite) TestLoadGlobalStats(c *C) {
-	defer cleanEnv(c, s.store, s.do)
-	tk := testkit.NewTestKit(c, s.store)
+func TestLoadGlobalStats(t *testing.T) {
+	t.Parallel()
+	tk, dom, clean := createTestKitAndDom(t)
+	defer clean()
 	tk.MustExec("use test")
 	tk.MustExec("set @@tidb_analyze_version = 2")
 	tk.MustExec("set @@tidb_partition_prune_mode = 'dynamic'")
 	tk.MustExec("drop table if exists t")
 	tk.MustExec("create table t (a int, key(a)) partition by hash(a) partitions 2")
 	tk.MustExec("insert into t values (1), (2)")
 	tk.MustExec("analyze table t")
-	globalStats := s.getStatsJSON(c, "test", "t")
+	globalStats := getStatsJSON(t, dom, "test", "t")
 
 	// remove all statistics
 	tk.MustExec("delete from mysql.stats_meta")
 	tk.MustExec("delete from mysql.stats_histograms")
 	tk.MustExec("delete from mysql.stats_buckets")
-	s.do.StatsHandle().Clear()
-	clearedStats := s.getStatsJSON(c, "test", "t")
-	c.Assert(len(clearedStats.Partitions), Equals, 0)
+	dom.StatsHandle().Clear()
+	clearedStats := getStatsJSON(t, dom, "test", "t")
+	require.Equal(t, 0, len(clearedStats.Partitions))
 
 	// load global-stats back
-	c.Assert(s.do.StatsHandle().LoadStatsFromJSON(s.do.InfoSchema(), globalStats), IsNil)
-	loadedStats := s.getStatsJSON(c, "test", "t")
-	c.Assert(len(loadedStats.Partitions), Equals, 3) // p0, p1, global
+	require.Nil(t, dom.StatsHandle().LoadStatsFromJSON(dom.InfoSchema(), globalStats))
+	loadedStats := getStatsJSON(t, dom, "test", "t")
+	require.Equal(t, 3, len(loadedStats.Partitions))
```

Comment:
Add the comment `// p0, p1, global` so it's easier for others to understand the test

Reply:
ok, updated